openshift/tests-extension/test/qe/specs/olmv1_cc.go

```diff
@@ -1,13 +1,20 @@
 package specs
 
 import (
+	"context"
```

Comment:
@jianzhangbjz FYI https://github.com/openshift/operator-framework-operator-controller/blob/main/openshift/tests-extension/test/qe/README.md#before-submitting-pr

need to run `make update-metadata` to update json file.